spec/core/cache_spec.rb

```diff
@@ -35,9 +35,63 @@ def initialize(opts = {})
     example_cache = File.expand_path(Fog::Cache.namespace(Fog::SubFogTestModel, @service)).downcase
     expected_namespace = File.expand_path("~/.fog-cache/for-service-user-region-foo").downcase
 
+    puts example_cache
```

Comment:
Could you remove these `puts`? I think I pulled them on master, as I presume they were just accidentally left behind?